### DISCUSSION TITLE
Show related position in file of posts

### DISCUSSION THREAD

**Discussion by Isaac-Leonard** [magit/forge#603] [Ideas]
Would it please be possible to show which file and where in the file is being referred to by a post in MRs.
If this is already possible please delete this but I couldn't find anything about it in the manual or by searching issues and discussions.

**Comment by tarsius** [comment]
Are you talking about "embedded code snippets" like this?:

https://github.com/magit/forge/blob/5dcca0e0c0378499b086543be2ec292ddd8beb5b/lisp/forge.el#L1

**Comment by Isaac-Leonard** [comment]
No sorry, I was thinking of something like what the code-review package does but I hadn't found that package yet when I posted this.

On 17 Nov 2023, at 09:25, Jonas Bernoulli ***@***.***> wrote:

﻿

Are you talking about "embedded code snippets" like this?:

https://github.com/magit/forge/blob/5dcca0e0c0378499b086543be2ec292ddd8beb5b/lisp/forge.el#L1
